tests/conftest.py

```diff
@@ -40,7 +40,7 @@
 from invenio_search import InvenioSearch
 
 
-@pytest.fixture()
+@pytest.yield_fixture()
```

Comment:
Actually ``yield_fixture`` is [deprecated](https://docs.pytest.org/en/latest/yieldfixture.html) since pytest v3, and you should just use ``fixture`` with a yield inside.

Reply:
The problem was with the `lowest` requirements Travis job, since `pytest v2.8.0`, does not allow `yield` statements if the `yield_fixture` decorator is not used: https://travis-ci.org/inveniosoftware/invenio-search/jobs/308961005#L889